src/test/java/core/basesyntax/service/impl/ReportServiceImplTest.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportService;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ReportServiceImplTest {
+    private ReportService reportService;
+
+    @BeforeEach
+    void setUp() {
+        reportService = new ReportServiceImpl();
+    }
+
+    @Test
+    void createReport_validData_ok() {
+        Storage.storage.put("banana", 20);
+        Storage.storage.put("apple", 80);
+        String expected = "fruit,quantity\nbanana,20\napple,80\n";
+        String actual = reportService.getReport();
+        Assertions.assertEquals(expected, actual);
+    }
+
+    @Test
+    void createReport_noTitle_notOk() {
+        Storage.storage.put("banana", 20);
```

Comment:
Also could be extracted to a constant,
```suggestion
        Storage.storage.put(BANANA, 20);
```